examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>(I'm gritting my teeth as I type, but can you begin with "We" when the step introduces a code block?)</p><p> </p><p>"We view the &lt;some-kind-of&gt; metrics that the <code>evaluate</code> method generated. The method produces a list of the metrics and the metrics are stored in the same order that they were generated."</p><p> </p><p>That's a little verbose, but if you can find a happy medium that mentions the name of the method, please do. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>